docs/user/metafeature_nucleotides_usage.md

```diff
@@ -0,0 +1,36 @@
+# Running `metafeature_nucleotides.R`
+
+`metafeature_nucleotides.R` is an R script that will produce a profile averaging the reads occuring around given nucleotide positions.
+
+### Inputs 
+
+`metafeature_nucleotides.R` requires 4 files as inputs:
+
+* A h5 file - Produced by riboviz, lists all of the reads and their positions relative to the start codon.
+* A gff file - For an example species, this file can be found in `example-datasets` on the riboviz github, in the `annotation` folder of the relevant species.
+* A fasta file - For an example species, this file can be found in `example-datasets` on the riboviz github, in the `annotation` folder of the relevant species.
+* A tsv file of nt positions to be studied - This file lists the gene under the heading 'Gene' and the nt postion under the heading 'Pos'. This is created by the user to fit their research question.
+
+### Outputs
+
+`metafeature_nucleotides.R` will output a PDF containing a metafeature plot showing the avergage number of reads occuring at each position in a desired window around the position of interest. It is averaged over each position listed in the tsv file. 
+
+### Arguments 
+
+The optional arguments are 
+
+```
+option_list <- list(make_option(c('-i', '--input'),type = "character", help='Path input to h5 file'),
+                    make_option(c('-d', '--dataset'),type = "character", help='Name of the dataset being studied'),
+                    make_option(c('-g', '--gff'),type = "character", help='Path to the GFF3 file of the organism being studied'),
+                    make_option(c('-f', '--fasta'),type = "character", help='Path to the fasta file of the organsim being studied'),
+                    make_option(c('--feature_pos'), type = "character", help='A TSV file listing the Gene and Positions to normalize over'),
```

Comment:
Referred to as tsv, not TSV earlier. 

For the "Gene" and "Positions" columns mentioned here, earlier you specify the heading is called "Pos" - could this cause confusion when making the tsv file and cause problems?